resources/home/dnanexus/utils/util_functions.py

```
@@ -0,0 +1,101 @@
+                    if snv_reports:
+                        if file.get("SNV count") == "0":
+                            file["snv_url"] = "No SNVs post-filtering"
+
+            # Do same for CNV reports, remove CNV report link if
+            # cnv_reports option True
+            if "CNV" in file_data:
+                for file in file_data["CNV"]:
+                    if file.get("CNV count") == "Unknown":
+                        file["CNV count"] = None
+                    if cnv_reports:
+                        if file.get("CNV count") == "0":
```

Ruff (0.8.0)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)